opus-sys/build.rs

```diff
@@ -15,14 +15,42 @@ fn format_write(builder: bindgen::Builder) -> String {
         .replace("/*!", "/*")
 }
 
+#[cfg(not(target_env = "msvc"))]
+fn try_vcpkg(_statik: bool) -> Option<Vec<PathBuf>> {
+    None
+}
+
+#[cfg(target_env = "msvc")]
+fn try_vcpkg(statik: bool) -> Option<Vec<PathBuf>> {
+    if !statik {
+        env::set_var("VCPKGRS_DYNAMIC", "1");
+    }
+
+    vcpkg::find_package("opus")
+        .map_err(|e| {
+            println!("Could not find ffmpeg with vcpkg: {}", e);
+        })
+        .map(|library| library.include_paths)
+        .ok()
+}
+
 fn main() {
-    let libs = metadeps::probe().unwrap();
-    let headers = libs.get("opus").unwrap().include_paths.clone();
+    let mut include_sub_dir = "";
```

Comment:
Is this subdirectory necessary on Windows? Can we find header files through clang in another way, setting an environment variable for example?

Reply:
I'm not sure about this. Really not familiar with build system.
Which I can figure out are follows :

1. In vcpkg file system, the header files of vpx, opus, aom are in their own folders, look like this:

> PS D:\Code\example\vcpkg\installed\x64-windows\include> tree /f
D:.
├─aom
│      aom.h
│      aomcx.h
│      aomdx.h
│      aom_codec.h
│      aom_decoder.h
│      aom_encoder.h
│      aom_external_partition.h
│      aom_frame_buffer.h
│      aom_image.h
│      aom_integer.h
│
├─opus
│      opus.h
│      opus_custom.h
│      opus_defines.h
│      opus_multistream.h
│      opus_projection.h
│      opus_types.h
│
└─vpx
        vp8.h
        vp8cx.h
        vp8dx.h
        vpx_codec.h
        vpx_decoder.h
        vpx_encoder.h
        vpx_ext_ratectrl.h
        vpx_frame_buffer.h
        vpx_image.h
        vpx_integer.h

2.  In the PR with vpx-rs project, you can see no include_sub_dir in the code. That is because the folder is specified in vpx-sys/data/vpx.h by

` #include <vpx/vpx_decoder.h>
#include <vpx/vp8dx.h>
#include <vpx/vpx_encoder.h>
#include <vpx/vp8cx.h> `

In aom-rs, it's the same ay vpx-rs.

3. In this opus-rs, howerve, the opus-sys/data/opus.h not contains the "opus" folder in the include path. So, this is the point.

After all, within the vcpkg system, both the three project may have same behaviour. I'm not sure which they look like in Linux.